docker/scripts/prometheus_targets.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+set -e
+
+# Generate file to merge into actual config file
+echo $PROMETHEUS_TARGETS | base64 -d >> /sidecars/etc/configured_targets.yml
+
+# Merge the two yaml files together, overwriting prometheus.yml in place
+yq m --inplace --arrays append /sidecars/etc/prometheus.yml /sidecars/etc/configured_targets.yml
+
+# Things for Cartel
```

Comment:
This makes this module very specific to Cartel, but lets merge and look for a refactor moment next year.